### ISSUE TITLE
Scala Common Enrich: extract event_vendor, event_schema, event_version

### ISSUE THREAD

**Issue by alexanderdean** [snowplow/snowplow#1007]
This is so that you can do aggregate queries (e.g. number of events of type X) using the enriched events TSV without having to drill into the `ue_properties` field.

Depends on #910 

See also: #1008, #1009

**Comment by alexanderdean** [comment]
Duplicate of #1800
